src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private final int bound = 100;
```

Comment:
The field 'bound' should be a constant. According to the style guide, constants must be written in uppercase and declared as 'static final'.